benchmarking_sbi/README.md

```diff
@@ -21,51 +20,49 @@ The subfolder `results/` contains csv files with results for the manuscript *Ben
 | `supplement_hyperparameters_snpe.csv`         | Hyperparameters for (S)NPE                          |
 | `supplement_hyperparameters_snre.csv`         | Hyperparameters for (S)NRE                          |
 
-
 ## Raw results for each run
 
 Raw results for each single run from which these dataframes were compiled—including samples from all approximate posteriors—are also available. For space reasons, they are hosted in a separately using LFS:
 
-> https://github.com/mackelab/benchmarking_sbi_runs
-
+> <https://github.com/mackelab/benchmarking_sbi_runs>
```

Comment:
I guess we don't need the `<>` tags -- shouldn't the URL be linked automatically?